projects/storefrontlib/src/cms-components/misc/icon/icon.component.ts

```diff
@@ -8,7 +8,9 @@ import {
 import { SafeHtml } from '@angular/platform-browser';
 import { DirectionMode } from '../../../layout/direction/config/direction.model';
 import { IconLoaderService } from './icon-loader.service';
-import { ICON_TYPE } from './icon.model';
+import { ICON_TYPE as DEFAULT_ICON_TYPE } from './icon.model';
+
+type ICON_TYPE = DEFAULT_ICON_TYPE | string;
```

Comment:
It's a breaking change. Icon type used to be an enum (so you can use its values i.e. `ICON_TYPE.STAR`), but now it becomes only a type.

Reply:
We should rather relax the typing in the places that refer to the ICON_TYPE, e.g. in the IconComponent (and perhaps other places - you need to check):
https://github.com/SAP/spartacus/blob/develop/projects/storefrontlib/src/cms-components/misc/icon/icon.component.ts#L42-L52

Reply:
My bad! I didn't pay attention it's the component file, I somehow thought it's the enum file. Sorry for confusion!